### DISCUSSION TITLE
how to  redirect to  different page in  react when puzzle  is solved

### DISCUSSION THREAD

**Discussion by wp-lohith** [flbulgarelli/headbreaker#71] [Q&A]
in  react how  to  redirect to  different page when puzzle is  solved

**Comment by flbulgarelli** [comment]
¡Hi! 

This library is independent of React, so there is no React-specific support for doing this. However, you could approach this by  combining `useEffect` and `useState`: declare a `setSolved` setter and a `solved` property using `useState`, and then use `useEffect` to assign an `onValid` callback, which will call `setSolved`. Finally, make your React component depend on `solved` so it displays the corresponding success page.
